packages/cli/src/commands/styles/utils/cssGenerator.utils.ts

```diff
@@ -0,0 +1,67 @@
+export const generateColorsCSS = (tokenMap: any) => {
+  const keyValuePairs = tokenMap.color.map((color: { token: string; value: string; }) => {
+    // eslint-disable-next-line
+    const key = `--ds-color${color.token.substring(8).trim().replaceAll("'", '')}`;
+    // eslint-disable-next-line
+    const value = color.value.replaceAll("'", '');
+    return [key, value];
+  });
+  return Object.fromEntries(keyValuePairs);
+};
+
+export const generateTypographyCSS = (tokenMap: any) => {
+  const keyValuePairs = tokenMap.typography.map((typography: { token: string; value: string; }) => {
+    // eslint-disable-next-line
+    const key = `--ds${typography.token.substring(1).trim().replaceAll("'", '')}`;
+    // eslint-disable-next-line
+    const value = typography.value.replaceAll("'", '');
+    return [key, value];
+  });
+  return Object.fromEntries(keyValuePairs);
+};
```

Comment:
It's a good thing its returning tuples, since that's the current format being returned from `template.spacing()`.

It might be a good idea to change the parameter to accept an array of `DesignToken[]` vs passing in a map

```export const generateTypographyCSS = (tokens: DesignToken[]) => {
        
      code
      
       return Object.fromEntries(keyValuePairs)
      } 
```
`['--ds-typography-font-weight', 'large']`
We should also create a type for the return type we have above... but we can do that after the demo :)